internal/common/run_yarn.js

```diff
@@ -1,5 +1,5 @@
 const child_process = require("child_process");
-const fs = require("fs");
+const path = require("path");
```

Comment:
While we're waiting for the skaffold issue to be addressed, it's probably worth merging this file as a separate PR?

Reply:
Sorry haven't gotten around to pulling this out. Let me get that real quick.